lib/src/position.dart

```diff
@@ -1086,8 +1168,10 @@ class Antichess extends Position<Antichess> {
 
   @override
   Outcome? get variantOutcome {
-    if (isVariantEnd || isStalemate) {
-      return Outcome(winner: turn);
+    if (isVariantEnd) {
+      return Outcome(winner: turn, endType: EndType.decisive);
+    } else if (isStalemate) {
+      return Outcome(winner: turn, endType: EndType.stalemate);
```

Comment:
stalemate can have a winner in chess sharp, but not in chess so it doesn't look right here. I guess we're missing tests for outcome in normal chess, or am I missing something?

Reply:
This section is in Antichess. The prior code suggested that stalemate is decisive.